### PR TITLE
Fix remaining tests

### PR DESCRIPTION
**Pull request by benjstephenson**

### CODE REVIEW

File: lib/functions/pipe/index.ts [hunk 1]
```diff
@@ -46,7 +46,7 @@ function pipe(
 		default:
 			const [head, ...rest] = arguments
 			return rest.reduce((result, fn) => fn(result), head)
-	}
-}
+	} /* v8 ignore next 3*/
+} /* v8 ignore next 3*/
```
Comment (chasm) [inline lib/functions/pipe/index.ts:50]: Well, that's one way. I could not figure out what was missing here that V8 thought should be tested. Maybe a V8 bug?

File: lib/functions/identity/index.test.ts [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
 
-// FIXME
 test("[identity] (functions) returns the identical item passed", () => {
-	// fc.assert(fc.property(fc.anything(), x => expect(identity(x)).toBe(x)))
+	fc.assert(
+		fc.property(fc.anything(), x => {
+			const result = identity(x)
+
+			if (x == null || x == undefined) expect(result).toEqual(x)
+
```
Comment (chasm) [inline lib/functions/identity/index.test.ts:13]: Doh.

File: lib/either/traverseAccumulate/index.ts [hunk 3]
```diff
@@ -25,7 +25,7 @@ const traverseAccumulate: TraverseAccumulate = concat => f => as => {
 
 		if (isLeft(acc) && isLeft(eitherB))
 			return left(concat(acc.left, eitherB.left))
-		else if (isLeft(acc)) return acc
+		/* v8 ignore next 3 */ else if (isLeft(acc)) return acc
 		else if (isLeft(eitherB)) return eitherB
```
Comment (chasm) [inline lib/either/traverseAccumulate/index.ts:29]: OK

**Review by chasm — APPROVED** [review]
Looks good to me.

File: lib/either/traverseAccumulate/index.ts (hunk 3 above)
Comment (benjstephenson) [reply lib/either/traverseAccumulate/index.ts:29]: Had a brainwave after I pushed, removed the ignore and added the test :)

File: lib/functions/pipe/index.ts (hunk 1 above)
Comment (benjstephenson) [reply lib/functions/pipe/index.ts:50]: So this I'm super confused about.  I don't understand why v8 is asking to test a brace....

**Pull request merged**